opencti-platform/opencti-front/tests_e2e/report/createReport.spec.ts

```diff
@@ -14,3 +15,19 @@ test('Create a new report page', async ({ page }) => {
   await reportPage.getItemFromList('Test e2e Unknown - admin No').click();
   await expect(reportDetailsPage.getReportDetailsPage()).toBeVisible();
 });
+
+test('Create a new report with associated file', async ({ page }) => {
+  const reportPage = new ReportPage(page);
+  const reportDetailsPage = new ReportDetailsPage(page);
+  const reportForm = new ReportFormPage(page);
+  await page.goto('/dashboard/analyses/reports');
+  await reportPage.addNewReport();
+  await reportForm.fillNameInput('Test e2e with file');
+  const fileChooserPromise = page.waitForEvent('filechooser');
+  await page.getByRole('button', { name: 'Select your file', exact: true }).click();
+  const fileChooser = await fileChooserPromise;
+  await fileChooser.setFiles(path.join(__dirname, 'createReport.spec.ts'));
+  await reportPage.getCreateReportButton().click();
+  await reportPage.getItemFromList('Test e2e with file').click();
+  await expect(reportDetailsPage.getReportDetailsPage()).toBeVisible();
```

Comment:
maybe add a test on the data tab > presence of the associated file in the list

Reply:
Done